cogs/welcome/welcome.py

```diff
@@ -470,6 +522,25 @@ async def greetings(self, ctx: Context):
         - `returning`: pool of greetings that are sent to returning users
         """
 
+    # [p]welcomeset greetings toggleimg
+    @greetings.command(name="toggleimg")
+    async def toggle_img(self, ctx: Context):
+        """Toggles the random image on and off"""
+        async with self.config.guild(ctx.guild).all() as guildData:
+            if guildData["toggle_img"]:
+                guildData["toggle_img"] = False
+            elif guildData["toggle_img"] == False:
+                # check if there is at least one image in the pool at least, otherwise tell user to add one before enabling
+                if len(os.listdir(self.img_dir)) < 1:
+                    await ctx.send(
+                        "There are currently no images in the image_base folder. Add at least one before turning the randomiser on"
```

Comment:
I don't think we need to tell the user about the specific folder. We just need to let them know there are no images for this functionality to work, and that they need to add (e.g., via `[p]welcomeset greetings addimg`) first.